server/athenian/api/openapi/openapi.yaml

```
@@ -4039,3 +4086,78 @@ components:
           description: New name of the token.
           type: string
       type: object
+    PullRequestHistogramRequest:
+      additionalProperties: false
+      description: Request to calculate a histogram over a pull request distribution.
+      properties:
+        for:
+          $ref: '#/components/schemas/ForSet'
+        id:
+          $ref: '#/components/schemas/PullRequestHistogramID'
+        scale:
+          $ref: '#/components/schemas/HistogramScale'
+        date_from:
+          description: Date from when to start measuring the distribution.
+          format: date
+          type: string
+        date_to:
+          description: Date up to which to measure the distribution.
+          format: date
+          type: string
+        timezone:
+          description: Local time zone offset in minutes, used to adjust `date_from`
+            and `date_to`.
+          type: integer
+        exclude_inactive:
+          description: Value indicating whether PRs without events in the given time
+            frame shall be ignored.
+          type: boolean
+        account:
+          description: Session account ID.
+          type: integer
+      required:
+      - account
+      - date_from
+      - date_to
+      - exclude_inactive
+      - for
+      - id
+      type: object
+    PullRequestHistogramID:
+      description: Supported histogram topics.
+      enum:
+      - pr-cycle-time
+      example: pr-cycle-time
+      type: string
+    HistogramScale:
+      description: X axis scale.
+      enum:
+      - linear
+      - log
+      example: log
+      type: string
+    CalculatedPullRequestHistogram:
+      additionalProperties: false
+      description: Calculated histogram definition.
+      properties:
+        x:
+          description: Series of horizontal bar borders. Their count is `len(y) + 1`
+            because there are `N` intervals between `(N + 1)` ticks.
+          items:
+# {% if False %}
+            allOf:
+# {% else %}{{ "\n" }}            oneOf:{% endif %}
+            - type: number
+            - format: date-time
+              type: string
+          type: array
+        y:
+          description: Series of histogram bar heights.
+          items:
+            type: integer
+          type: array
+      required:
+      - xmin
+      - xmax
+      - bars
```

This does not match the defined `CalculatedPullRequestHistogram` properties.
I'd also rename the properties. `x` and `y` are not that _histogram terms_, wdyt?
```suggestion
      - ticks # or also `thresholds`
      - values
```

---

Also, if the `scale` is not part of the `PullRequestHistogramRequest` (because it will be optional), I'd send it in the `CalculatedPullRequestHistogram`.